apps/web/src/pages/activities/components/ActivityList.tsx

```diff
@@ -33,27 +34,34 @@ export function ActivityList({ data: userData, pagination = false, loading = fal
           return <ActivityItem onClick={onRowClick} key={`activity-item-${item.id}`} item={item} />;
         })}
       </div>
-      {pagination && total > 0 && pageSize > 1 && getPageCount() > 1 && (
-        <Pagination
-          styles={{
-            item: {
-              marginTop: '15px',
-              marginBottom: '15px',
-              backgroundColor: 'transparent',
-              '&[data-active]': {
-                backgroundImage: colors.horizontal,
-                border: 'none',
-              },
-            },
-          }}
-          total={getPageCount()}
-          page={current + 1}
-          onChange={(pageNumber) => {
-            onPageChange(pageNumber - 1);
-          }}
-          position="center"
-        />
-      )}
+
+      <div
+        style={{
+          marginTop: '10px',
+          display: 'flex',
+          width: '100%',
+          justifyContent: 'center',
+          alignItems: 'center',
+          padding: '15px',
+        }}
+      >
+        <Button.Group>
+          <Button
+            variant="outline"
+            disabled={pagination?.current === 0 || loading}
+            onClick={() => onPageChange(pagination?.current - 1)}
+          >
+            <ChevronLeft />
+          </Button>
+          <Button
+            variant="outline"
+            disabled={!pagination?.hasMore || loading}
+            onClick={() => onPageChange(pagination?.current + 1)}
+          >
+            <ChevronRight />
+          </Button>
```

Comment:
If you use Button from our design system it would looks: 
<img width="179" alt="Screen Shot 2023-07-16 at 13 18 06" src="https://github.com/novuhq/novu/assets/7778680/bbcdaeaf-cc20-40ee-8a7a-07eaa3b4bff3">
instead of:
<img width="181" alt="Screen Shot 2023-07-16 at 13 18 31" src="https://github.com/novuhq/novu/assets/7778680/45fab00e-142d-442c-b3b5-5e5fc1818e2f">